schemas/user/summary.yml

```diff
@@ -0,0 +1,33 @@
+properties:
+  id:
+    type: string
+    readOnly: true
+    description: |
+      The user's identifier (generated by the API)
+
+  url:
+    type: string
+    readOnly: true
+    description: |
+      API url for accessing the user description.
+
+  email:
+    type: string
+    format: email
+    readOnly: true
+    description: |
+      The email address used when authenticating the user.
```

Comment:
Wasn't sure whether to use email or username here. Both would work, but wondering how likely it is that we will be making use of the email for anything other than authentication, given the way numi will be deployed and used.

Since the plan is for Numi to be deployed in country for a single organisation, rather than a cloud-based platform, I'm not sure if emailing users is something we actually want to do. Password resets would probably be done by admins rather than an email-based process.We might at some point want to send notification emails to people, but that approach seems like it would fit a cloud-based solution better than some internal Numi instance deployed for some organisation.

Thoughts?